test/API/H5_api_group_test.c

```diff
@@ -1728,14 +1728,14 @@ test_get_group_info(void)
 
                 if (group_info.nlinks != 0) {
                     H5_FAILED();
-                    HDprintf("    group's number of links '%lld' doesn't match expected value '%lld'\n",
+                    HDprintf("    group's number of links '%zu' doesn't match expected value '%d'\n",
```

Comment:
Since this is an `hsize_t` value, we should use the form `" ... links '%" PRIuHSIZE "' doesn't ... "`

Reply:
Got it. I can do that.